pkgs/development/python-modules/mashumaro/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, setuptools
+, typing-extensions
+, orjson
+, msgpack
+, pyyaml
+, tomli-w
+, tomli
+, pytestCheckHook
+, ciso8601
+, pendulum
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "mashumaro";
+  version = "3.8.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Fatal1ty";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-WDKohmcdVlQR/6AMSISN0y6UQx4tmOf1fANCPLRYiqI=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    typing-extensions
+  ];
+
+  passthru.optional-dependencies = {
+    orjson = [ orjson ];
+    msgpack = [ msgpack ];
+    yaml = [ pyyaml ];
+    toml = [ tomli-w ] ++ lib.optionals (pythonOlder "3.11") [ tomli ];
+  };
+
+  nativeCheckInputs = [
+    ciso8601
+    msgpack
+    orjson
+    pendulum
+    pytest-mock
+    pytestCheckHook
+    pyyaml
+    tomli-w
+  ] ++ lib.optionals (pythonOlder "3.11") [
+    tomli
+  ];
+
+  pythonImportsCheck = [
+    "mashumaro"
+  ];
+
+  meta = with lib; {
```

Comment:
Would you please add the changelog to the meta?

Reply:
Of course, I added:
```
changelog = "https://github.com/Fatal1ty/mashumaro/releases/tag/v${version}";
```